sdk/helper/storagepacker/types.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+option go_package = "github.com/hashicorp/vault/sdk/helper/storagepacker";
+
+package storagepacker;
+
+import "google/protobuf/any.proto";
+
+// Item represents an entry that gets inserted into the storage packer
+message Item {
```

Comment:
Given that v2 is using raw bytes instead of this Item message it seems like we don't need this type and all of the related functions at all?

Reply:
Item is still the return type, but it doesn't need to be a Protobuf type anymore. I can change that.